app/src/main/java/com/hypertrack/views/quickstart/MainActivity.java

```diff
@@ -30,15 +30,15 @@ protected void onResume() {
         super.onResume();
         Log.d(TAG, "onResume: ");
 
-        mHyperTrackData = HyperTrackViews.getInstance(this, PUBLISHABLE_KEY);
-        mHyperTrackData.getDeviceMovementStatus("A4005505-D469-481E-AD2E-123456789AB",
+        mHyperTrackView = HyperTrackViews.getInstance(this, PUBLISHABLE_KEY);
+        mHyperTrackView.getDeviceMovementStatus("A4005505-D469-481E-AD2E-123456789AB",
                 new Consumer<MovementStatus>() {
                     @Override
                     public void accept(MovementStatus movementStatus) {
                         Log.d(TAG, "Got movement status data " + movementStatus);
                     }
                 });
-        mHyperTrackData.subscribeToDeviceUpdates("A4005505-D469-481E-AD2E-123456789AB",
+        mHyperTrackView.subscribeToDeviceUpdates("A4005505-D469-481E-AD2E-123456789AB",
```

Comment:
maybe we can make the device_id a property and add a comment next to pointing out it should be replaced (same for PUBLISHABLE_KEY)?

Reply:
Done!